src/js/views/Project/Project.jsx

```diff
@@ -214,8 +207,7 @@ function Project() {
     setDocumentTitle(t('common.notFound'))
     return <Error>{t('error.notFound')}</Error>
   } else if (state.project === null || state.loading) {
```

Comment:
Just remove the whole loading state display and remove L209 and 210 in your code, unless the ProjectPage blows up without it.

Reply:
It does blow up (needs factTypes for example)